_components/accordions.md

```diff
@@ -144,13 +144,13 @@ title: Accordions
 }
 
 #accordion-menu ul li.hidden p {
-  background: url("../img/arrow-right.png") no-repeat;
+  background: url("https://raw.githubusercontent.com/18F/govt-wide-patternlibrary/18f-pages-staging/assets/img/arrow-right.png") no-repeat;
```

Comment:
You might want to use the production URL as described here:

https://rawgit.com/

This will ensure that customers of the library won't have this break when due to throttling or blacklisting.